frontend/src/app/tasks/month/page.tsx

```diff
@@ -1,3 +1,134 @@
-export default function MonthView() {
-  return <div>Month!</div>
+import {
+  startOfMonth,
+  startOfWeek,
+  lastDayOfMonth,
+  lastDayOfWeek,
+  format,
+  eachDayOfInterval,
+  isSameDay,
+  isSameMonth
+} from 'date-fns'
+import { Box, Typography } from '@mui/joy'
+import { getTasksWithDate } from '../actions/getTasksWithDate'
+import { convertMinutesToTime } from '../utils/time'
+import { SimpleTaskBox } from '../components/TaskBox'
+import { Fragment } from 'react'
+
+const WEEK_DAYS = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+
+export default async function MonthView() {
+  const today = new Date()
+  const firstDayOfFirstWeek = startOfWeek(startOfMonth(today))
+  const lastDayOfLastWeek = lastDayOfWeek(lastDayOfMonth(today))
+  const dateRange = eachDayOfInterval({ start: firstDayOfFirstWeek, end: lastDayOfLastWeek })
+
+  const groupedTasks = await getTasksWithDate(
+    format(firstDayOfFirstWeek, 'yyyy-MM-dd'),
+    format(lastDayOfLastWeek, 'yyyy-MM-dd'),
+    500
+  )
+
+  return (
+    <Box
+      sx={{
+        display: 'grid',
+        gridTemplateColumns: 'repeat(7, 1fr)'
+      }}
+      alignItems="stretch"
+      height="100%"
+    >
+      {WEEK_DAYS.map((day, index) => {
+        const isToday = index === today.getDay()
+        return (
+          <Typography
+            sx={{
+              textAlign: 'center',
+              padding: { xs: '8px', sm: '22px 16px' },
+              borderBottom: '1px solid #C4C6D0',
+              borderTop: '1px solid #C4C6D0',
+              borderRight: '1px solid #C4C6D0',
+              ':first-of-type': { borderLeft: '1px solid #C4C6D0' },
+              ':nth-of-type(7n)': { borderRight: { sm: 'none' }, borderLeft: { sm: 'none' } },
+              fontWeight: '600',
+              color: isToday ? '#004c92' : 'auto',
+              fontSize: { xs: '0px', sm: '1rem' },
+              ':first-letter': { fontSize: '1rem' }
+            }}
+            component="div"
+            key="day"
+          >
+            {isToday ? 'Today' : day}
```

Comment:
I am not sure we want the whole day of the week column to show as 'Today' instead of 'Wednesday'. I know the mockup shows this, but I think it is enough to have the slightly darker background for the day that is today. 
![Screenshot from 2024-01-24 10-59-44](https://github.com/Igalia/phpreport/assets/3666105/17504bb1-c39a-4924-902a-4f44dbd4d8ba)